pkg/volume/plugins.go

```diff
@@ -121,6 +122,9 @@ type VolumeHost interface {
 	// the provided spec.  See comments on NewWrapperBuilder for more
 	// context.
 	NewWrapperCleaner(spec *Spec, podUID types.UID, mounter mount.Interface) (Cleaner, error)
+
+	//Get cloud provider from kubelet
+	GetCloudProvider() cloudprovider.Interface
```

Comment:
The VolumeHost is is the software abstraction that hosts the plugin, not the machine.  Plugins should have access to only the code they link themselves and the code we offer through the host interface.  If this is something we think plugins should be able to do ("give me a pointer to the current CloudProvider"), the the Host interface is the correct place.